### DISCUSSION TITLE
[BUG]中文前缀的域名部署后无法正常接收邮件

### DISCUSSION THREAD

**Discussion by eomly** [dreamhunter2333/cloudflare_temp_email#398] [General]
## 复现步骤
将中文域名（如：秦始皇.eu.org）正常添加到CF，后端设置变量：DOMAINS=秦始皇.eu.org时，用邮箱给xxx@秦始皇.eu.org发邮件时无法接收。
但是若后端DOMAINS设置为”xn--exsq28d6jc.eu.org“（对应”秦始皇.eu.org“），用邮箱给xxx@秦始皇.eu.org发邮件时可以接收邮件。

## 预期行为
因为后端DOMAINS设置为”xn--exsq28d6jc.eu.org“，在网页中也同样是显示”xn--exsq28d6jc.eu.org“，能否在网页显示为中文的”秦始皇.eu.org“？或者从根本上解决这个问题，即后端设”秦始皇.eu.org“时也能正常接收邮件。

## 部署方式

- [ ]  用户界面部署

## 浏览器环境
Google

**Comment by dreamhunter2333** [comment]
感谢反馈，对中文域名的工作方式不太了解，不过你提供的第二种方案 "在网页显示为中文" 比较好实现，可以在创建邮件和网页顶部显示成中文域名，不过邮件中的地址就不太好解决了

**Comment by eomly** [comment]
> 感谢反馈，对中文域名的工作方式不太了解，不过你提供的第二种方案 "在网页显示为中文" 比较好实现，可以在创建邮件和网页顶部显示成中文域名，不过邮件中的地址就不太好解决了

感谢大佬，期待更新给个教程。

**Comment by dreamhunter2333** [comment]
main分支已增加，之后会发一个版本, 使用 main 可以参考此处的文档

https://github.com/dreamhunter2333/cloudflare_temp_email/blob/881e66e484047cb459b1fbf3228f4c25b348fe7d/vitepress-docs/docs/zh/guide/cli/worker.md?plain=1#L57C1-L58C41

**Comment by eomly** [comment]
> main分支已增加，之后会发一个版本, 使用 main 可以参考此处的文档
> 
> https://github.com/dreamhunter2333/cloudflare_temp_email/blob/881e66e484047cb459b1fbf3228f4c25b348fe7d/vitepress-docs/docs/zh/guide/cli/worker.md?plain=1#L57C1-L58C41

感谢大佬！

**Comment by dreamhunter2333** [comment]
https://github.com/dreamhunter2333/cloudflare_temp_email/releases/tag/v0.5.3

v0.5.3 已增加
